### PR TITLE
Fix for lowecase tablenames. Fixes issue:14

### PR DESCRIPTION
**Pull request by berkes**
This fixes issue #14

### CODE REVIEW

**Comment by mockdeep** [comment]
Looks like you've got a merge conflict of some sort.

**Comment by berkes** [comment]
@mockdeep how so? I cannot see the conflict, care to help me?

**Comment by mockdeep** [comment]
I'm seeing the grey bar saying "This pull request cannot be automatically merged."  Have you merged the latest from master into your branch?

**Comment by berkes** [comment]
On 13-07-12 15:47, Robert Fletcher wrote:

> I'm seeing the grey bar saying "This pull request cannot be automatically merged."  Have you merged the latest from master into your branch?

I had, yes. The reason that github shows the message is probably because
the pull-request stems from a local feature-branch. The commit, however,
can be pulled, cherry-picked or merged with git, just fine. AFAIK

Bèr

## 

  http://berk.es

  Open Source Webdevelopment.
  Ruby on Rails development.
  Drupal Development and -consultancy.

**Comment by mockdeep** [comment]
Weird.  I've never seen it done that way.  At any rate, it's been merged.  Closing.

**Comment by berkes** [comment]
Thanks! I will refine my pull-request system a little, so Github knows how to handle andy future pull-reqs here. -i.e: merge back feature-branches first

File: app/models/user.rb [hunk 1]
```diff
@@ -740,14 +740,14 @@ def belongs_to_projects
   
   #returns list of recent projects with a max count
   def recent_projects(max = 10)
-    project_ids = ActivityStream.find_by_sql("SELECT project_id FROM Activity_Streams WHERE actor_id = #{self.id} AND updated_at > '#{Time.now.advance :days => (Setting::DAYS_FOR_RECENT_PROJECTS * -1)}' GROUP BY project_id ORDER BY MAX(updated_at) DESC LIMIT #{max}").collect {|a| a.project_id}.join(",")
+    project_ids = ActivityStream.find_by_sql("SELECT project_id FROM activity_streams WHERE actor_id = #{self.id} AND updated_at > '#{Time.now.advance :days => (Setting::DAYS_FOR_RECENT_PROJECTS * -1)}' GROUP BY project_id ORDER BY MAX(updated_at) DESC LIMIT #{max}").collect {|a| a.project_id}.join(",")
```
Comment (brickthorn) [inline app/models/user.rb:743]: Here's some test `ruby` code:

``` ruby
def something
  puts "something"
end
```

Cool!

Comment (brickthorn) [inline app/models/user.rb:743]: Here's another block of `ruby` code:

``` ruby
def my_method
  puts "does something"
end
```

**Pull request closed without merging**